blob/drivertest/drivertest.go

```diff
@@ -14,26 +14,6 @@
 //
```

Comment:
Please turn this into a blank line, otherwise the whole license header will be copied into the godoc.

Reply:
Done.